ec_grasp_planner/src/planner.py

```diff
@@ -200,26 +204,57 @@ def handle_run_grasp_planner(self, req):
 
         return plan_srv.RunGraspPlannerResponse(ha.xml())
 
+
+def getParam(obj_type_params, obj_params, paramKey):
+    param = obj_type_params.get(paramKey)
+    if param is None:
+        param = obj_params.get(paramKey)
+    return param
+
+
 # ================================================================================================
 def create_surface_grasp(object_frame, support_surface_frame, handarm_params, object_type):
 
-    # Get the relevant parameters for hand object combination
-    if (object_type in handarm_params['surface_grasp']):            
-        params = handarm_params['surface_grasp'][object_type]
-    else:
-        params = handarm_params['surface_grasp']['object']
+    # # Get the relevant parameters for hand object combination
+    # if (object_type in handarm_params['surface_grasp']):
+    #     params = handarm_params['surface_grasp'][object_type]
+    # else:
+    #     params = handarm_params['surface_grasp']['object']
+
+    obj_type_params = {}
+    obj_params = {}
+    if (object_type in handarm_params['surface_grasp']):
+        obj_type_params = handarm_params['surface_grasp'][object_type]
+    if 'object' in handarm_params['surface_grasp']:
+        obj_params = handarm_params['surface_grasp']['object']
+
+    # hand_transform = params['hand_transform']
```

Comment:
the old code was good, so please revert the `getParam()` related changes. 
In general old code "in commented section" should not be part of a pull request to master!

Reply:
I agree on the old code "in commented section" part in your comment. 
The `getParam()` part is a bit surprising, since you committed this change in commit 699fe19c5060fe74cdd3e54c56b29627db72ed18 , so I assumed this is intended and had a good reason. I can revert it if you changed your mind. Just give another quick "keep" or "revert"